packages/block-library/src/embed/test/index.native.js

```diff
@@ -816,4 +816,36 @@ describe( 'Embed block', () => {
 
 		expect( cannotEmbedText ).toBeDefined();
 	} );
+
+	it( 'insert embed from slash inserter', async () => {
+		const embedBlockSlashInserter = '/Embed';
+		const { getByPlaceholderText, getByA11yLabel } = await initializeEditor(
+			{
+				initialHtml:
+					'<!-- wp:paragraph --><p></p><!-- /wp:paragraph -->',
+			}
+		);
+
+		const paragraphText = getByPlaceholderText( 'Start writing…' );
+		fireEvent( paragraphText, 'focus' );
+		fireEvent( paragraphText, 'onChange', {
+			nativeEvent: {
+				eventCount: 1,
+				target: undefined,
+				text: embedBlockSlashInserter,
+			},
+		} );
+
+		fireEvent.press(
```

Comment:
That's a great idea, I'm wondering if we could query by the `Embed` text instead of the a11y label, as the items in the slash inserter display a text label, wdyt?

<img width="428" alt="Screenshot 2021-11-02 at 14 09 15" src="https://user-images.githubusercontent.com/14905380/139852806-16b85df5-763e-4a5c-bdd8-452a8050e13b.png">

Reply:
I applied this suggestion in [this commit](https://github.com/WordPress/gutenberg/pull/35533/commits/3b7faa9bcba257dee471936d462db6888284bd8b).